lib/ProxySQL_Admin.cpp

```diff
@@ -632,8 +633,26 @@ using admin_gauge_tuple =
 		metric_tags
 	>;
 
+using admin_dyn_counter_tuple =
+	std::tuple<
```

Comment:
This maybe personal preference, but I would think that these instead could be made into a base type, as most of the members are the same. Perhaps a template struct, which takes an enum as the template parameter for the first member and constructor. Then for these other using declarations, a single line type alias would suffice like `using admin_dyn_counter = metric_t<p_admin_dyn_counter>`. This would also make compiler error messages easier to understand than if the `make_tuple` call was incorrect. Then, rather than std::get with index, access by member name, easier to read and less error prone if the data structure changed. For example, if another value was added to the tuple as the first param, would invalidate the indices used by std::get calls. Whereas, member access would not have to be rewritten. While the implementation here appears functional, it might not be necessary to refactor or redo this, however should be kept in mind for future implementations.